src/main.rs

```diff
@@ -81,17 +81,17 @@ pub struct Opt {
 }
 #[derive(StructOpt, Debug, Serialize, Deserialize, PartialEq)]
 pub enum OptAction {
-    #[structopt(name = "update", help = "update the value of a variable")]
+    #[structopt(name = "update", help = "update the value of a variable, in a running eww instance")]
     Update { fieldname: VarName, value: PrimitiveValue },
 
-    #[structopt(name = "open", help = "open a window")]
+    #[structopt(name = "open", help = "opens a window")]
```

Comment:
This is inconsistent with the rest of the help messages
```suggestion
    #[structopt(name = "open", help = "open a window")]
```